docs/source/tutorial/cli/convert.rst

```diff
@@ -0,0 +1,10 @@
+.. _convert:
+
+Conversion
+----------
+
+To convert a trained model from Chemprop v1 to v2, run ``chemprop convert`` and specify:
+
+ * :code:`--input-path <path>` Path of the Chemprop v1 file to convert.
+ * :code:`--output-path <path>` Path where the converted Chemprop v2 will be saved. If unspecified, this will default to ``<current working directory>/<stem of input>_v2.ckpt``.
```

Comment:
Is there a way to point these to the CLI code? If we update the arguments, we will also need to remember to update this here. In #675 I changed the style of the default from `<current working directory>/<stem of input>_v2.ckpt` to `PATH/TO/V1_MODEL_v2.ckpt` to be stylistically consistent with the other default paths in the CLI.

If there isn't a way, to do this updating automatically, then it will be the responsibility of future PRs (like #675) to update the docs as well, which isn't the end of the world.

Reply:
I'm not sure of how to do this automatically for these pages that we write ourselves, but I think everything will be updated automatically for things in the "CLI Reference" section. I just updated this path to be consistent with yours for now.